custom_components/vicare/binary_sensor.py

```diff
@@ -27,14 +27,38 @@
 SENSOR_CIRCULATION_PUMP_ACTIVE = "circulationpump_active"
 SENSOR_BURNER_ACTIVE = "burner_active"
 SENSOR_COMPRESSOR_ACTIVE = "compressor_active"
-
+SENSOR_FROST_PROTECTION_ACTIVE = "frost_protection_active"
+SENSOR_DHW_CIRCULATION_PUMP_ACTIVE = "dhw_circulation_pump_active"
+SENSOR_CHARGING_ACTIVE = "charging_active"
+SENSOR_DHW_PUMP_ACTIVE = "dhw_pump_active"
 
 @dataclass
 class ViCareBinarySensorEntityDescription(
     BinarySensorEntityDescription, ViCareRequiredKeysMixin
 ):
     """Describes ViCare binary sensor entity."""
 
+GLOBAL_SENSORS: tuple[ViCareBinarySensorEntityDescription, ...] = (
+    ViCareBinarySensorEntityDescription(
+        key=SENSOR_DHW_CIRCULATION_PUMP_ACTIVE,
+        name="DHW Circulation Pump Active",
+        device_class=DEVICE_CLASS_POWER,
+        value_getter=lambda api: api.getDomesticHotWaterCirculationPumpActive(),
+    ),
+    ViCareBinarySensorEntityDescription(
+        key=SENSOR_CHARGING_ACTIVE,
+        name="Charging Active",
```

Comment:
we should rename it to "Hot Water Charging Active" to be clear

Reply:
Makes sense. We could also use DHW Charging Active as DHW is used in a few other places. How about that?

Reply:
This one done